config/default.yml

```
@@ -237,6 +237,16 @@ Rails/EnvironmentComparison:
   Enabled: true
   VersionAdded: '0.52'
 
+Rails/EnvironmentVariableAccess:
+  Description: 'Do not access `ENV` directly after initialization.'
+  Enabled: true
```

Can you change to `pending` status?

```suggestion
  Enabled: pending
```